README.md

```diff
@@ -158,17 +158,14 @@ helm chart export $CHART_REF --destination $CHART_EXPORT_PATH
       cd -
       ```
   - [Option 2] Namespace scoped deployment
-    - Specify the namespace to listen to
-      ```sh
-      export WATCH_NAMESPACE=<NAMESPACE_TO_LISTEN_TO>
-      ```
+    - It will listen to the namespace that you specify when running helm install [3.1.3](#313-install-controller). In this sample that will be the value ACK_K8S_NAMESPACE is set to.
```

Comment:
The controller will watch for the resources in the helm chart release namespace. In this guide, that value is set from the `$ACK_K8S_NAMESPACE` variable in helm install section [3.1.3](#313-install-controller)